src/Test.UnitTests.Sarif.Multitool.Library/RebaseUriCommandTests.cs

```diff
@@ -27,6 +27,7 @@ public class RebaseUriCommandTests : FileDiffingUnitTests
         public RebaseUriCommandTests(ITestOutputHelper outputHelper) : base(outputHelper) { }
 
         [Fact]
+        [Trait(TestTraits.WindowsOnly, "true")]
```

Comment:
>WindowsOnly [](http://example.com/codeflow?start=26&length=11)

Why are these tests Windows only? Can you explain why they can't run on Unix platforms?